docs/modules/ROOT/partials/permissive/model-expressions/given.adoc

```diff
@@ -1,12 +1,27 @@
 === `+GIVEN+`
 
+.Compute the probability of the satellite's period from the model, given the class of orbit.
 [example]
 ====
 [source,gensql]
 ----
 SELECT
   PROBABILITY OF Period_minutes
-  UNDER satellites_model GIVEN Class_of_Orbit
+  UNDER satellites_model
+  GIVEN Class_of_Orbit
+FROM satellites
+----
+====
+
+.Compute the probability of the satellite's period from the model, given everything but the contractor and operator's country.
+[example]
+====
+[source,gensql]
+----
+SELECT
+  PROBABILITY OF Period_minutes
+  UNDER satellites_model
+  GIVEN * EXCEPT Contractor, Country_of_Operator
```

Comment:
Can we change that to include `Period_minutes` in the `EXCEPT` - not conditioning on the target is the most important use case, in a sense.